draft-thomson-httpbis-http2bis.xml

```diff
@@ -3163,9 +3163,9 @@
         </t>
         <t>
           Promised requests MUST be cacheable (see <xref target="RFC7231" section="4.2.3"/>), MUST be safe
-          (see <xref target="RFC7231" section="4.2.1"/>), and MUST NOT include a request body. Clients that receive a
+          (see <xref target="RFC7231" section="4.2.1"/>), and MUST NOT include any content. Clients that receive a
```

Comment:
Separately, I'm wondering if this "MUST NOT" is better as "cannot" to match the "is not possible" below.

Reply:
Hmm, which "is not possible"? I don't see an is-not-possible anywhere near here.

Reply:
> It is not possible to push a response to a request that includes message content.

In the next bit of diff.  Not as close in the file though.

Reply:
Ah, I see. My mistake for looking through the file. :wink: But I see what you're driving at.

I think we need at least one normative statement that promised requests must not contain content. Otherwise, saying "promised requests cannot include any content" invites the question "well, why not?". Having at least one normative statement means the answer is "because we said so" and that's a sufficiently good answer for spec purposes. So I'm inclined to leave it.